package.json

```
@@ -29,9 +31,13 @@
 		"dotenv": "^16.3.1"
 	},
 	"devDependencies": {
+		"@commitlint/cli": "^19.8.1",
+		"@commitlint/config-conventional": "^19.8.1",
 		"@types/node": "^20.11.17",
+		"husky": "^9.1.7",
 		"nodemon": "^3.0.0",
 		"prettier": "2.7.1",
+		"prettier-plugin-asciidoc": "^0.0.1-prerelease-1",
 		"typescript": "^5.3.3"
 	}
-}
+}
```

```suggestion
}

```

Please add a new line at the end of the file.

---

We should add an `.editorconfig` file for this.
https://editorconfig.org/#overview